guides/development-workflow.md

```diff
@@ -190,6 +190,7 @@ To deploy:
   ```
   git fetch
   git checkout master
+  git pull
```

Comment:
@tahb is that only because you have `--no-ff` on by default? 

Do you agree that we only ever want to merge develop into the latest version of master (the one on origin) - if so how can we do that without pulling?